gui/widgets/matRad_WorkflowWidget.m

```diff
@@ -403,11 +433,11 @@ function btnOptimize_Callback(this, hObject, eventdata)
                 
                 pln = evalin('base','pln');
                 ct  = evalin('base','ct');
-                
+                dij = evalin('base','dij');
                 % optimize
-                [resultGUIcurrentRun,usedOptimizer] = matRad_fluenceOptimization(evalin('base','dij'),evalin('base','cst'),pln);
+                [resultGUIcurrentRun,usedOptimizer] = matRad_fluenceOptimization(dij,evalin('base','cst'),pln);
                 if pln.propOpt.conf3D && strcmp(pln.radiationMode,'photons')
-                    resultGUIcurrentRun.w = resultGUIcurrentRun.w * ones(dij.totalNumOfBixels,1);
+                    resultGUIcurrentRun.w = resultGUIcurrentRun.w .* ones(dij.totalNumOfBixels,1); % what and why ? 
                     resultGUIcurrentRun.wUnsequenced = resultGUIcurrentRun.w;
                 end
                 
```

Comment:
There is dead code in lines 464-483 that can probably be deleted.

Reply:
cleared